Source/m_menu.c

```diff
@@ -3301,6 +3303,9 @@ setup_menu_t enem_settings1[] =  // Enemy Settings screen
   // [FG] colored blood and gibs
   {"Colored Blood",S_YESNO,m_null,E_X,E_Y+ enem_colored_blood*8, {"colored_blood"}},
 
+  // [crispy] randomly flip corpse, blood and death animation sprites
```

Comment:
Does this fit on a menu page?

Reply:
Yes, "Number Of Single-Player Helper Dogs" is even longer.